### PR TITLE
"Configuring Your Inventory File" section sync with v3.11

### PR DESCRIPTION
**Pull request by bysnupy**
Fix the issue: https://github.com/openshift/openshift-docs/issues/12467

I found this page in `master` was older than `v3.11` same one, so I replace the page with `v3.11` one.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@mburke5678, PTAL

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -246,13 +250,13 @@ on an Amazon Web Services (AWS) cluster. The default value is `false`.
...
-defaults `{"region": "infra"}`. See
+defaults `{"node-role.kubernetes.io/infra":"true"}`. See
 xref:configuring-node-host-labels[Configuring Node Host Labels] for details.
 
 |`osm_default_node_selector`
 |This variable overrides the node selector that projects will use by default when
 placing pods, which is defined by the `projectConfig.defaultNodeSelector` field
-in the master configuration file. This defaults
+in the master configuration file. Starting in {product-title} 3.9, this defaults
 to `node-role.kubernetes.io/compute=true` if undefined.
```
Comment (mburke5678) [inline install/configuring_inventory_file.adoc]: @bysnupy The _Starting in {Product-title} 3.9_ was removed through https://github.com/openshift/openshift-docs/commit/4168b2abe0c3e08caa43dee903ace30d9bba67bf#diff-80f7c8ee1abcb1ceb42d8350b49bf604. 

The sentence starts with _This defaults..._

**Comment by mburke5678** [comment]
@bysnupy @kalexand-rh All of Daein's changes are present in the 3.11 version of this document and not in the master version, with one exception, due to a recent change.

**Comment by kalexand-rh** [comment]
Thank you @mburke5678!

File: install/configuring_inventory_file.adoc (hunk 1 above)
Comment (kalexand-rh) [reply install/configuring_inventory_file.adoc]: Fixing here: https://github.com/openshift/openshift-docs/pull/12510

**Pull request merged**